reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-70.md

```diff
@@ -450,17 +450,22 @@ using those, check out our [PowerShell-Docker](https://hub.docker.com/_/microsof
 
 ### Running PowerShell 7
 
-PowerShell 7 runs side-by-side with previous versions of Windows PowerShell or PowerShell Core. When
+PowerShell 7 runs side-by-side with previous versions of Windows PowerShell. When
```

Comment:
Adding the install folder to the path does not always occur (eg with DailyBuild).  I thought that you needed to use the MSI for that. When I install the daily build that folder is NOT added to my path variable.

I see this;
```powershell
PS C:\foo> .\Install-PWSH7.ps1 -Destination c:\dailybuild -daily
VERBOSE: Destination: C:\dailybuild
VERBOSE: About to download package from 'https://pscoretestdata.blob.core.windows.net/v7-0-0-daily-20200211/PowerShell-7.0.0-daily.20200211-win-x64.zip'
VERBOSE: Change icon to disambiguate it from a released installation
PowerShell has been installed at C:\dailybuild
PS C:\foo> $env:path.split(';')
C:\Program Files\PowerShell\7-preview
C:\WINDOWS\system32
C:\WINDOWS
C:\WINDOWS\System32\Wbem
C:\WINDOWS\System32\WindowsPowerShell\v1.0\
C:\WINDOWS\System32\OpenSSH\
C:\Program Files\Microsoft VS Code\bin
C:\Program Files (x86)\Calibre2\
C:\Program Files\dotnet\
C:\Program Files\PowerShell\7-preview\preview
C:\Program Files\Git\cmd
C:\Users\tfl.COOKHAM\AppData\Local\Microsoft\WindowsApps
C:\Users\tfl.COOKHAM\.dotnet\tools
C:\Users\tfl.COOKHAM\AppData\Local\Microsoft\WindowsApps
C:\windows\Microsoft.NET\Framework\v4.0.30319\
PS C:\foo>
```